src/main/scala/tech/cryptonomic/conseil/routes/openapi/TezosEndpoints.scala

```diff
@@ -80,15 +80,10 @@ trait TezosEndpoints extends algebra.Endpoints with DataJsonSchemas with ApiFilt
       tags = List("Operation groups")
     )
 
-  /** Average fees endpoint definition */
-  def avgFeesEndpointV1: Endpoint[(String, ApiOperations.Filter, String), Option[AverageFees]] =
-    endpoint(
-      request = get(
-        url = commonPath / "operations" / "avgFees" /? qsFilter,
-        headers = header("apiKey")),
-      response = jsonResponse[AverageFees](docs = Some("Endpoint for average fees")).orNotFound(Some("Not found")),
-      tags = List("Fees")
-    )
+  /** Operation groups row schema */
+  /** Pasted here because somehow it is not being pulled in from DataJsonSchemas at all. */
+  override implicit def operationsRowSchema: JsonSchema[OperationsRow] =
```

Comment:
For some reason, `operationsRowSchema()` from `DataJsonSchemas` does not get pulled into the implicit context here. Removing this line causes errors and I cannot figure out why `operationGroupsRowSchema()` works and `operationsRowSchema()` does not. Any ideas, @piotrkosecki @dorians and @ivanopagano ?

Reply:
I've investigated it and I think error message is wrong - it should be rather `ambiguous implicit`. There is already implicit for `OperationsRow`(`implicit def operationSchema: JsonSchema[OperationsRow]`, line 68), I removed it locally and it works. I saw that you used the new it explicitly in the encoder, so you can remove the old one, also it was poorly named, so thats why you didn't find it.

Reply:
Wow, I am not very impressed with the Scala compiler!